service/src/main/java/com/appirio/service/billingaccount/resources/BillingAccountResource.java

```diff
@@ -112,11 +113,22 @@ public ApiResponse searchBillingAccounts(@Auth AuthUser user,
      */
     @POST
     @Path("billing-accounts")
-    public ApiResponse createBillingAccount(@Auth AuthUser user, @Valid BillingAccount request) {
+    public ApiResponse createBillingAccount(@Auth AuthUser user, @Valid PostPutRequest<BillingAccount> request) {
         try {
             checkAdmin(user);
-            return MetadataApiResponseFactory
-                    .createResponse(billingAccountManager.createBillingAccount(user, request));
+            if (request.getMethod() == null) {
```

Comment:
@bannie2492 according to the v3 spec it must be provided? if not let's make it default to post.

how about `patch`?